br/README.md

```diff
@@ -1,131 +1,190 @@
+For example, backup raw data in TiKV cluster to s3 `/backup-data/2022-09-16` directory.
 
-```sh
-# Using tiup to start a TiDB cluster
-tiup playground --db 2 --pd 3 --kv 3 --monitor
+```
+export AWS_ACCESS_KEY_ID=&{AWS_KEY_ID};
+export AWS_SECRET_ACCESS_KEY=&{AWS_KEY};
+tikv-br backup raw \
+    --pd "&{PDIP}:2379" \
+    -s "s3://backup-data/2022-09-16/" \
+    --ratelimit 128 \
+    --dst-api-version v2 \
+    --log-file="/tmp/br_backup.log
+```
+Explanations for some options in the above command are as follows: 
+- `backup`: Sub-command of `tikv-br`.
+- `raw`: Sub-command of `backup`.
+- `-s` or `--storage`: Storage of backup files.
+- `"s3://backup-data/2022-09-16/"`: Parameter of `-s`, save the backup files in `"s3://backup-data/2022-09-16/"`.
+- `--ratelimit`: Specifies the maximum speed at which a backup operation is performed on each `TiKV` node.
+- `128`: The value of `ratelimit`, unit is MiB/s.
+- `--pd`: Service address of `PD`.
+- `"${PDIP}:2379"`:  Parameter of `--pd`.
+- `--dst-api-version`: The `api-version`, please see [tikv-server config](https://docs.pingcap.com/tidb/stable/tikv-configuration-file#api-version-new-in-v610).
+- `v2`: Parameter of `--dst-api-version`, the optionals are `v1`, `v1ttl`, `v2`(Case insensitive). If no `dst-api-version` is specified, the `api-version` is the same with TiKV cluster of `--pd`.
+
+A progress bar is displayed in the terminal during the backup. When the progress bar advances to 100%, the backup is complete. The progress bar is displayed as follows:
+```
+br backup raw \
+    --pd "${PDIP}:2379" \
+    --storage "s3://backup-data/2022-09-16/" \
+    --log-file backupfull.log
+Backup Raw <---------/................................................> 17.12%.
+```
 
-# Using tiup bench to generater test data.
-tiup bench tpcc --warehouses 1 prepare
+After backup finish, the result message is displayed as follows:
+```
+[2022/09/20 18:01:10.125 +08:00] [INFO] [collector.go:67] ["Raw backup success summary"] [total-ranges=3] [ranges-succeed=3] [ranges-failed=0] [backup-total-regions=3] [total-take=5.050265883s] [BackupTS=436120585518448641] [total-kv=100000] [total-kv-size=108.7MB] [average-speed=21.11MB/s] [backup-data-size(after-compressed)=78.3MB]
+```
+Explanations for the above message are as follows: 
+- `total-ranges`: specifies the ranges' count that backup separate the task. Equals to `ranges-succeed` + `ranges-failed`.
+- `ranges-succeed`: specifies the succeeded task count.
+- `ranges-failed`: specifies the failed task count.
+- `backup-total-regions`: specifies the tikv regions that backup takes.
+- `total-take`: specifies the backup duration.
+- `BackupTS`: specifies the backup start timestamp, only take effect for APIV2 TiKV cluster, which can be used as TiKV-CDC [start-ts](https://github.com/tikv/migration/blob/main/cdc/README.md).
```

Comment:
The last timestamp ?
"which can be used as `start-ts` of `TiKV-CDC` when creating replication tasks. Refer to [`Create a replication task`](https://github.com/tikv/migration/blob/main/cdc/README.md#create-a-replication-task).